src/azure-cli/azure/cli/command_modules/netappfiles/_help.py

```diff
@@ -917,6 +921,28 @@
 """
 
 
+helps['netappfiles volume backup status'] = """
+type: group
+short-summary: See Azure NetApp Files (ANF) Volume Backup Status.
+"""
+
+helps['netappfiles volume backup status show'] = """
```

Comment:
If `netappfiles volume backup status` only have a `show` command, you should use `netappfiles volume backup show-status`

Reply:
But if status will have `create` or `update` commands in the future. That's fine to add `netappfiles volume backup status` command group

Reply:
@kairu-ms I agree with you but volume backup status came in a separate command file in the SDK. I have not seen a way to be able to include two command types in one command group. This will probably be have to be fixed in our swagger. I will take a look at this.

Reply:
@kairu-ms I removed everything regarding this volume backup status operation because this will be moved under backups operations in the next swagger merge.